extensions/wikia/Forum/ForumHooksHelper.class.php

```diff
@@ -1,6 +1,38 @@
 <?php
 
 class ForumHooksHelper {
+
+	static public function discussionNavigationOverride(
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused function parameters "$title, $wallMessage, $path, $request". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1172)